cli/src/main.rs

```diff
@@ -109,7 +129,7 @@ async fn main() -> Result<(), color_eyre::Report> {
     }
 
     let config = iroha::combine_configs(&args)?;
-    let telemetry = iroha_logger::init(&config.logger)?;
+    let telemetry = iroha_logger::init_global(&config.logger, terminal_colors)?;
```

Comment:
This is no longer telemetry, but a logger handle, probably needs renaming

Reply:
Fixed